WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueNotificationsContentView.swift

```diff
@@ -19,7 +19,7 @@ struct UnifiedPrologueNotificationsContentView: View {
                 RoundRectangleView {
                     HStack {
                         NotificationIcon(image: Appearance.topImage, size: notificationIconSize)
-                        Text(string: Appearance.topElementTitle)
+                        Text(Appearance.topElementTitle)
```

Comment:
I was having build errors when trying to build that from the Jetpack target.

I removed the `string: ` and it seems to work from both WordPress and Jetpack. But perhaps I missed something, so it would be good if you can double-check @Gio2018 

Thanks in advance!

Reply:
Ah, I think we need to add Text+BoldSubString.swift to the Jetpack target (and then re-add this init parameter!) 🙂

Reply:
Done!